### PR TITLE
Fixes failure in test_dataloader.py that occurs on jetson boards

### PR DESCRIPTION
**Pull request by puririshi98**
CUDA IPC is not supported for jetsons

### CODE REVIEW

File: test/test_dataloader.py [hunk 1]
```diff
@@ -67,7 +67,8 @@
 # as well during the execution of this test suite, and it will cause
 # CUDA OOM error on Windows.
 TEST_CUDA = torch.cuda.is_available()
-
+dev_name = torch.cuda.get_device_name(torch.cuda.current_device()).lower()
```
Comment (ptrblck) [inline test/test_dataloader.py:70]: Would this crash, if no GPU is available, or what is `cuda.get_device_name` returning in this case?

File: test/test_dataloader.py [hunk 2]
```diff
@@ -1335,7 +1336,7 @@ def test_multiprocessing_contexts(self):
         dl_common_args = dict(num_workers=3, batch_size=3, pin_memory=(not TEST_CUDA))
         for ctx in supported_multiprocessing_contexts:
             # windows doesn't support sharing cuda tensor; ROCm does not yet fully support IPC
-            if ctx in ['spawn', 'forkserver'] and TEST_CUDA and not IS_WINDOWS:
+            if ctx in ['spawn', 'forkserver'] and TEST_CUDA and not IS_WINDOWS and not IS_JETSON:
```
Comment (ptrblck) [inline test/test_dataloader.py:1339]: Maybe update the comment, too.

File: test/test_dataloader.py (hunk 1 above)
Comment (puririshi98) [reply test/test_dataloader.py]: yes it would crash, i added a guard against this.

**Review by ejguan — APPROVED** [review]
LGTM, thanks for adding this patch.

**Pull request closed without merging**